docs(react-grid): add information about using Virtual Table with formatters (#2505)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/guides/virtual-scrolling.md

If the Grid should have the same size as the container element, set the `VirtualTable` plugin's `height` property to "auto" and the Grid root element's style setting to `height: 100%`.

.embedded-demo({ "path": "grid-virtual-scrolling/stretching-to-parent-element", "showThemeSelector": true })

## Note on the use of `VirtualTable` with `DataTypeProvider` and custom components

> If you use a custom `rowComponent` or `cellComponent`, their height and the `estimatedRowHeight` value should be equal. The same applies to a custom formatter defined in the [DataTypeProvider](../reference/data-type-provider.md) plugin.

Reviewer: Change "their height" to "its height"

Reviewer 2: ```suggestion
If you use a custom `rowComponent` or `cellComponent`, its height and the `estimatedRowHeight` value should be equal. The same applies to a custom formatter defined in the [DataTypeProvider](../reference/data-type-provider.md) plugin.
```

> The virtual table renders only visible rows. Empty space before and after rows is calculated using `estimatedRowHeight` property. Default `estamatedRowHeight` property is equal default row height. If you use [DataTypeProvider](../reference/data-type-provider.md) plugin to define custom formatter, you have to be sure that `estimatedRowHeight` property is equal to new row height. The same applies to the custom cell and row components.

Reviewer 2: ```suggestion
If you use a custom `rowComponent` or `cellComponent`, their height and the `estimatedRowHeight` value should be equal. The same applies to a custom formatter defined in the [DataTypeProvider](../reference/data-type-provider.md) plugin.
```
